### ISSUE TITLE
Damage * Defense rating?  DmgD?

### ISSUE THREAD

**Issue by Durpady** [fribbels/Fribbels-Epic-7-Optimizer#70]
This would be handy for units who scale damage off of Defense, like DLilibet, FMaya, and if you wanted to use her that way, Rose.

**Comment by fribbels** [comment]
I think eventually the app should move towards either customizeable scores, or per-unit damage calcs. Low prio for now

**Comment by Durpady** [comment]
Makes sense, especially with the wonky multipliers for units that can scale off of Speed.
